src/nimble/NimbleBluetooth.cpp

```
@@ -220,6 +280,25 @@ class BluetoothPhoneAPI : public PhoneAPI, public concurrency::OSThread
   protected:
     virtual int32_t runOnce() override
     {
+        if (pendingNimbleInitRetry) {
+            const uint32_t retryDelayMs = nimbleInitRetryDelayMs(nimbleInitRetryCount);
+            const uint32_t remainingMs = Throttle::remainingTimespanMs(nimbleInitRetryStartedAtMs, retryDelayMs, millis());
```

Clang (14.0.6)

[warning] variable 'remainingMs' is not initialized

(cppcoreguidelines-init-variables)